agents1/GenericAgent.py

```diff
@@ -1,19 +1,20 @@
 import sys
-from typing import final, List, Dict, Final
-import enum, random
-from bw4t.BW4TBrain import BW4TBrain
-from matrx.agents.agent_utils.state import State
-from matrx.agents.agent_utils.navigator import Navigator
-from matrx.agents.agent_utils.state_tracker import StateTracker
+from typing import Dict
+
 from matrx.actions.door_actions import OpenDoorAction
 from matrx.actions.object_actions import GrabObject, DropObject
-from matrx.messages.message import Message
+from matrx.agents.agent_utils.navigator import Navigator
+from matrx.agents.agent_utils.state import State
+from matrx.agents.agent_utils.state_tracker import StateTracker
+
+from agents1.Message import MessageBuilder as mb, MessageType as mt
```

Comment:
I would recommend renaming this to something more readable

Reply:
on it! 🚀🧑‍🚀